platform-ajax/src/main/java/com/softicar/platform/ajax/document/creation/AjaxDocumentPagePrinter.java

```diff
@@ -118,8 +118,6 @@ private void printJavascriptUpdateCode() {
 		println("<script>");
 		println("GLOBAL_init();");
 		updateCodeWriter.apply();
-		println("lock(LOCK_REASON_DOM_EVENT);");
-		println("ACTION_QUEUE.executeNextAction();");
```

Comment:
I tested this, the call to `ACTION_QUEUE.executeNextAction()` does nothing since the queue is always empty on document creation. This is of course to be expected. I don't know why this was called here. Its removal also makes the `lock` call unnecessary, or rather impossible.